package.json

```diff
@@ -1,27 +1,50 @@
 {
-  "name": "ds-toolbox-test",
-  "version": "0.0.4",
-  "description": "A demo repository to illustrate how to share palette and typography through NPM",
-  "license": "MIT",
-  "private": false,
-  "files": [
-    "/assets",
-    "/tasks"
-  ],
+  "name": "@texastribune/queso-ui",
+  "version": "0.0.1-0",
+  "description": "Asset library of SCSS and SVG files",
   "scripts": {
-    "lint": "stylelint 'assets/**/**.scss' && prettier --write './tasks/**/*.js'",
-    "deploy-docs": "npm run build-docs && surge ./docs/dist --domain 'https://ttds.surge.sh/'",
-    "build-docs": "node docs/build/build.js",
-    "dev-docs": "node docs/build/develop.js"
+    "lint": "stylelint 'assets/**/**.scss' && prettier --write './docs/**/*.js' && eslint docs/config",
```

Comment:
Looks like the stylelint and prettier stuff is accounted for as a pre-commit hook; could this script just have eslint (like it does in queso-tools)?

Reply:
I like the idea of being able to do it manually, without invoking a commit. I can make only eslint happen on lint though, if it bugs ya, and put the others in their own command.

Reply:
Nah that's fine. Do whatever suits you!